lib/commander/runner.rb

```diff
@@ -304,7 +304,7 @@ def create_default_commands
           if args.empty?
             say help_formatter.render
           else
-            command = command args.join(' ')
+            command = command(valid_command_names_from(*args.dup).sort.last)
```

Comment:
This new logic was copied from `command_name_from_args` method.

https://github.com/commander-rb/commander/blob/b8abd42153118ca9fdc02affeb9f3aab61cda5a2/lib/commander/runner.rb#L241-L243

There's undoubtedly an opportunity to refactor to DRY this up. Unfortunately, I haven't the time today.

Reply:
I've extracted a new method to DRY this up.